### ISSUE TITLE
训练乒乓球，振荡很大

### ISSUE THREAD

**Issue by 121649982** [YonghaoHe/A-Light-and-Fast-Face-Detector-for-Edge-Devices#74]
你好，训练时loss变化很大，这个正常吗？

2020-02-25 10:26:47,227[INFO]: cls_loss_score_1: --> 47.5173
2020-02-25 10:26:47,227[INFO]: reg_loss_bbox_1: --> 3581.6255
2020-02-25 10:26:47,227[INFO]: cls_loss_score_2: --> 555.4069
2020-02-25 10:26:47,227[INFO]: reg_loss_bbox_2: --> 1146.7589
2020-02-25 10:26:47,228[INFO]: cls_loss_score_3: --> 217.9907
2020-02-25 10:26:47,228[INFO]: reg_loss_bbox_3: --> 2087.8438
2020-02-25 10:26:47,228[INFO]: cls_loss_score_4: --> 1833.2731
2020-02-25 10:26:47,228[INFO]: reg_loss_bbox_4: --> 2937.1301
2020-02-25 10:26:47,228[INFO]: total loss = 10927.7227
2020-02-25 10:26:56,808[INFO]: Iter[8300] -- Time elapsed: 9.6 s. Speed: 41.8 images/s.
2020-02-25 10:26:56,808[INFO]: cls_loss_score_0: --> 56.8907
2020-02-25 10:26:56,860[INFO]: reg_loss_bbox_0: --> 3565.3206
2020-02-25 10:26:56,860[INFO]: cls_loss_score_1: --> 44.3509
2020-02-25 10:26:56,860[INFO]: reg_loss_bbox_1: --> 1188.9268
2020-02-25 10:26:56,860[INFO]: cls_loss_score_2: --> 52.6698
2020-02-25 10:26:56,860[INFO]: reg_loss_bbox_2: --> 633.9600
2020-02-25 10:26:56,860[INFO]: cls_loss_score_3: --> 321.2800
2020-02-25 10:26:56,860[INFO]: reg_loss_bbox_3: --> 3745.3579
2020-02-25 10:26:56,860[INFO]: cls_loss_score_4: --> 817.3853
2020-02-25 10:26:56,861[INFO]: reg_loss_bbox_4: --> 1543.6974
2020-02-25 10:26:56,861[INFO]: total loss = 9335.2256
2020-02-25 10:27:06,472[INFO]: Iter[8400] -- Time elapsed: 9.6 s. Speed: 41.6 images/s.
2020-02-25 10:27:06,473[INFO]: cls_loss_score_0: --> 47.5426
2020-02-25 10:27:06,519[INFO]: reg_loss_bbox_0: --> 1374.2328
2020-02-25 10:27:06,519[INFO]: cls_loss_score_1: --> 131.5045
2020-02-25 10:27:06,519[INFO]: reg_loss_bbox_1: --> 1402.3698
2020-02-25 10:27:06,519[INFO]: cls_loss_score_2: --> 45.1156
2020-02-25 10:27:06,519[INFO]: reg_loss_bbox_2: --> 1253.8846
2020-02-25 10:27:06,519[INFO]: cls_loss_score_3: --> 263.4740
2020-02-25 10:27:06,519[INFO]: reg_loss_bbox_3: --> 1571.5750
2020-02-25 10:27:06,520[INFO]: cls_loss_score_4: --> 1505.2943
2020-02-25 10:27:06,520[INFO]: reg_loss_bbox_4: --> 1666.0612
2020-02-25 10:27:06,520[INFO]: total loss = 8881.0312
2020-02-25 10:27:16,390[INFO]: Iter[8500] -- Time elapsed: 9.9 s. Speed: 40.5 images/s.
2020-02-25 10:27:16,390[INFO]: cls_loss_score_0: --> 1354.4061
2020-02-25 10:27:16,442[INFO]: reg_loss_bbox_0: --> 3272.2812
2020-02-25 10:27:16,442[INFO]: cls_loss_score_1: --> 57.2416
2020-02-25 10:27:16,442[INFO]: reg_loss_bbox_1: --> 1328.8054
2020-02-25 10:27:16,443[INFO]: cls_loss_score_2: --> 85.8663
2020-02-25 10:27:16,443[INFO]: reg_loss_bbox_2: --> 2408.4707
2020-02-25 10:27:16,443[INFO]: cls_loss_score_3: --> 670.4828
2020-02-25 10:27:16,443[INFO]: reg_loss_bbox_3: --> 1462.6251
2020-02-25 10:27:16,443[INFO]: cls_loss_score_4: --> 524.8186
2020-02-25 10:27:16,443[INFO]: reg_loss_bbox_4: --> 825.2332
2020-02-25 10:27:16,443[INFO]: total loss = 19072.2773

**Comment by YonghaoHe** [comment]
@121649982 这个看起来有点不正常。按照我的判断，乒乓球的检测应该很简单，很快就会收敛。比如车牌检测的收敛速度就远远快于人脸检测。
